### PR TITLE
LMB-617 | Dynamic fractie options from LMB-640 | LATEST

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Dynamic fractie options in selector

## How to test

Create/update/correct a mandataris for a person in two bestuursperiodes

## Links to other PR's

- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/290

## Attachments

image

### CODE REVIEW

File: app/services/fractie-api.js [hunk 1]
```diff
@@ -41,4 +48,19 @@ export default class FractieApiService extends Service {
...
+        if (!type) {
+          return null;
+        }
+
+        return type.uri !== FRACTIETYPE_ONAFHANKELIJK ? fractie : null;
+      })
+    );
+
+    return samenWerkendeFracties.filter((fractie) => fractie);
+  }
```
Comment (JonasVanHoof) [inline app/services/fractie-api.js:65]: Add this to the resource call with a filter

Comment (JonasVanHoof) [reply app/services/fractie-api.js]: Done

**Comment by JonasVanHoof** [comment]
Cleanup on cancel of form to*⚠️

**Comment by JonasVanHoof** [comment]
- Random change of bestuursperiode in organen select?
- legislatuur always showing all fracties?

**Review by Rahien — APPROVED** [review]

**Pull request merged**